modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -1,7 +1,8 @@
-:description: Examples of how to manage constraints used for ensuring data integrity.
-:page-toclevels: 1
 [[constraints-examples]]
 = Examples
+:description: Examples of how to manage constraints used for ensuring data integrity.
+:page-toclevels: 1
+include::{ga-courses-uri}/cypher-indexes-constraints/ad.adoc[]
```

Comment:
In the preview, this goes here: https://graphacademy.neo4j.com/courses/app-go/?ref=docs-ad-go - shouldnt it go to the Indexes & Constraints course?